### PR TITLE
feat(appeals): function pipelines / initial consumer functions

### PR DESCRIPTION
**Pull request by fabfab-kc**
New azure function base code for the following integration consumers:

- Appellant case import (data produced by FO)
- LPAQ import (data produced by FO)
- HR import (data produced by ODW)

BOAT-498, BOAT-499, BOAT-500

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/functions/hr-data-import/local.settings.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"IsEncrypted": false,
+	"Values": {
+		"FUNCTIONS_WORKER_RUNTIME": "node",
+		"AzureWebJobsStorage": "UseDevelopmentStorage=true",
+		"API_HOST": "http://localhost:3000"
```
Comment (shepda) [inline appeals/functions/hr-data-import/local.settings.json:6]: Should API_HOST contain the protocol as the POST request has it hardcoded to https?

File: appeals/functions/casedata-import/local.settings.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+	"IsEncrypted": false,
+	"Values": {
+		"FUNCTIONS_WORKER_RUNTIME": "node",
+		"AzureWebJobsStorage": "UseDevelopmentStorage=true",
+		"API_HOST": "http://localhost:3000"
```
Comment (shepda) [inline appeals/functions/casedata-import/local.settings.json:6]: Should API_HOST contain the protocol as the POST request has it hardcoded to https?

File: appeals/functions/lpaq-import/local.settings.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+	"IsEncrypted": false,
+	"Values": {
+		"FUNCTIONS_WORKER_RUNTIME": "node",
+		"AzureWebJobsStorage": "UseDevelopmentStorage=true",
+		"API_HOST": "http://localhost:3000"
```
Comment (shepda) [inline appeals/functions/lpaq-import/local.settings.json:6]: Should API_HOST contain the protocol as the POST request has it hardcoded to https?

File: appeals/functions/hr-data-import/local.settings.json (hunk 1 above)
Comment (fabfab-kc) [reply appeals/functions/hr-data-import/local.settings.json:6]: That's a local settings only, will be replaced by the deployment pipelines... A bit pointless as testing end-to-end not really an option, so the functions are just a msg forwarder to the API.

**Review by shepda — APPROVED** [review]
LGTM

**Pull request merged**